Core/src/main/java/com/paypal/android/core/graphql/common/Utils.kt

```diff
@@ -0,0 +1,18 @@
+package com.paypal.android.core.graphql.common
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+fun JSONArray?.toStringsList(): List<String> {
+    return if (this == null) emptyList()
+    else {
+        MutableList(this.length()) {
+            this.getString(it)
+        }
+    }
+}
+
+fun JSONObject.getJSONArrayOrNull(key: String): JSONArray? {
+    return if (has(key)) this.getJSONArray(key)
+    else null
+}
```

Comment:
This is not needed, you can use `jsonObject.optJsonArray()` which does exactly this.
```suggestion
```

Reply:
it should return `JSONArray?` as per [documentation](https://developer.android.com/reference/org/json/JSONObject#optJSONArray(java.lang.String))

Reply:
@jcnoriega it's not returning `JSONArray?` it might be one of those java to kotlin conversion things,
it needed `@Nullable` in java to make it `JSONArray?`

Reply:
yeah i just checked and you are right!